OpenOversight/app/main/views.py

```diff
@@ -950,6 +950,26 @@ def download_incidents_csv(department_id):
     return Response(csv, mimetype="text/csv", headers=csv_headers)
 
 
+@main.route('/export/departments')
+def export_departments():
+    departments = {}
+    for d in Department.query.order_by(Department.id).all():
+        departments[d.id] = {**d.toCustomDict(), 'officers': []}
+
+    images = {}
+    for face in Face.query.order_by(Face.id).all():
+        if face.officer_id not in images:
+            images[face.officer_id] = []
+        images[face.officer_id].append(face.image.filepath)
```

Comment:
Should this actually build the s3 links? Something like:

 face_paths = []
        for face in faces:
            face_paths.append(serve_image(face.image.filepath))